jaxrs-clients/src/main/java/com/palantir/remoting2/jaxrs/JaxRsClient.java

```diff
@@ -35,21 +34,21 @@ private JaxRsClient() {}
      * ServiceName (Version)}, e.g. MyServer (1.2.3) For services that run multiple instances, recommended user agents
      * are of the form: {@code ServiceName/InstanceId (Version)}, e.g. MyServer/12 (1.2.3).
      */
-    public static <T> T create(Class<T> serviceClass, String userAgent, ClientConfiguration serviceConfig) {
-        return new FeignJaxRsClientBuilder(serviceConfig).build(serviceClass, userAgent);
+    public static <T> T create(Class<T> serviceClass, String userAgent, ClientConfiguration config) {
```

Comment:
I wouldn't want both on the classpath and would prefer to not do to reflecting stuff to figure out what's there